app/src/main/java/justforcommunity/radiocom/activities/Gallery.java

```diff
@@ -25,33 +25,26 @@
 import android.graphics.Color;
 import android.os.Bundle;
 import android.support.v4.view.ViewPager;
-import android.support.v4.view.WindowCompat;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.app.AppCompatDelegate;
 import android.support.v7.widget.Toolbar;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
-import android.view.Window;
 import android.view.WindowManager;
 
 import com.google.android.gms.analytics.GoogleAnalytics;
 import com.google.android.gms.analytics.HitBuilders;
 import com.google.android.gms.analytics.Tracker;
 import com.google.gson.Gson;
 
-import java.util.List;
-
 import justforcommunity.radiocom.R;
 import justforcommunity.radiocom.adapters.StationPhotosPagerAdapter;
 import justforcommunity.radiocom.model.StationDTO;
 import justforcommunity.radiocom.utils.GlobalValues;
 
 
-/**
- * Created by iver on 5/9/16.
- */
-public class Gallery extends AppCompatActivity{
+public class Gallery extends AppCompatActivity {
```

Comment:
it is important? 
I was only in some classes that you created, not at all, so it is homogeneous.